__main__.py

```diff
@@ -67,7 +67,15 @@ def load_map(self):
 
         self.object_layers = utils.process_objects(f"assets/tilemaps/TestLevel.tmx")
 
-        self.guard1_locations = utils.extract_guard_locations(self.object_layers[0])
+        self.guard_locations = [
+            utils.extract_locations(self.object_layers["guard"][i])
+            for i in range(len(self.object_layers["guard"]))
+        ]
```

Comment:
Both these loops can be done more pythonically
```suggestion
        self.guard_locations = [
            utils.extract_locations(guard)
            for guard in self.object_layers["guard"])
        ]
```
not sure if guard is the best name, but something like this